src/validate.py

```diff
@@ -0,0 +1,33 @@
+# SPDX-FileCopyrightText: © 2024 Kevin Lu
```

Comment:
This file definitely should be yours :)

Reply:
The blame is good enough for me. Transferring the copyright to you.